src/blitter/32bpp_sse_func.hpp

```diff
@@ -438,6 +438,12 @@ void Blitter_32bppSSSE3::Draw(Blitter::BlitterParams *bp, BlitterMode mode, Zoom
 void Blitter_32bppSSE4::Draw(Blitter::BlitterParams *bp, BlitterMode mode, ZoomLevel zoom)
 #endif
 {
+	if (_screen_disable_anim) {
+		/* This means our output is not to the screen, so we must ignore any translucency stuff */
+		Draw<BM_NORMAL, RM_WITH_MARGIN, BT_NONE, false>(bp, zoom);
+		return;
+	}
+
```

Comment:
This looks potentially problematic. _screen_disable_anim is not only used for drawing the touchbar sprites.
Drawing of sprites still needs to be correct when it is true.

This does not handle `mode`, `bp->skip_left`, `bp->width`, etc. and it's perfectly reasonable to render translucent sprites in a screenshot for example.

Reply:
The fix is inspired by the 40bpp-anim blitter, although that doesn't seem to have the alpha channel problem. What do you suggest?
https://github.com/OpenTTD/OpenTTD/blob/e5203f34092903693276b76f5c09bf00530e025f/src/blitter/40bpp_anim.cpp#L316-L320.

Reply:
That's a different switch, 40bpp changes the blitter, while you change the blitting mode.